recipe/meta.yaml

```diff
@@ -56,6 +61,7 @@ outputs:
         - cmake
       host:
         - zlib
+        - pcre2
```

Comment:
I am a bit unsure but we might want to use `pcre2-static` here then?

Reply:
Hm, let's give it a try!

```suggestion
        - pcre2-static
```